LetsDoStuff.WebApi/Controllers/ActivityController.cs

```diff
@@ -26,11 +26,6 @@ public ActionResult<List<ActivityResponse>> GetActivities()
         {
             var activities = _activityService.GetAllActivities();
 
-            if (activities == null)
-            {
-                return NotFound();
-            }
-
```

Comment:
Why you delete this?

Reply:
Because activities never could be null. Method GetAllActivities always returns a list and if our DB is empty then we'll get an empty list.

Reply:
It's a refactoring for tl004, not for tl007

Reply:
> It's a refactoring for tl004, not for tl007

When I noticed this bug I asked Konstantin about fixing this by myself and I fixed this imminently when he'd said: "yes". But in highlight, I understand that creating an extra branch with the refactoring task would be a better idea here. Sorry about this. The next time I will do this the right way 👍